zonemaster/zonemaster-gui
# remove dependency on moment

## Purpose

Remove the dependency on moment.js as the library as stopped development (and that we don't use it that much anyway).

## Context

#275

## Changes

* Parse date using standard javascript instead of moment.

## How to test this PR

* Navigate though the GUI, if no error are shown in the console and that nothing looks out of place it should be good.
